docs/pull_requests.md

```diff
@@ -0,0 +1,30 @@
+# Pull Requests
```

Comment:
@LindsayYoung This doc is fantastic! Thank you for writing it up and encoding all the practices we've been discussing. 😄